### DISCUSSION TITLE
raylib issue, difficulties running the library

### DISCUSSION THREAD

**Discussion by Lumiconic** [microsoft/vscode-discussions#1047] [Extension Development QnA] [answered]
Hi,

There's this awesome library called raylib, but making it run in vscode is nearly impossible. There is a template, but it doesn't work anymore. No tutos work for adding this library too anymore. The author of the library told me that it was very difficult to get it to work, when he created the template some years ago. Since the library is a very cool tool for dev, it would be great if vscode devs could look in to it.

**Comment by gjsjohnmurray** [comment, marked as answer]
Are you trying to use raylib to **build an extension** for VS Code? If not, this isn't the right forum for you. Please see the [READ FIRST](https://github.com/microsoft/vscode-discussions/discussions/5) post which is pinned to the top of the forum. It contains links to other resources.
